include/boost/geometry/geometries/concepts/check.hpp

```diff
@@ -39,7 +39,7 @@
 #include <boost/geometry/geometries/concepts/polygon_concept.hpp>
 #include <boost/geometry/geometries/concepts/ring_concept.hpp>
 #include <boost/geometry/geometries/concepts/segment_concept.hpp>
-
+#include <boost/geometry/geometries/concepts/PolyhedralSurface_concept.hpp>
```

Comment:
Please, do not use CamelCase in file names (i.e. `PolyhedralSurface_concept.hpp` should read `polyhedralsurface_concept.hpp`